src/client/South/SouthStatus.jsx

```diff
@@ -6,10 +6,14 @@ import PointsButton from './PointsButton.jsx'
 import { ConfigContext } from '../context/ConfigContext.jsx'
 import Table from '../components/table/Table.jsx'
 
-function isIsoDate(str) {
-  if (!/\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}Z/.test(str)) return false
-  const d = new Date(str)
-  return d.toISOString() === str
+function formatValue(value) {
```

Comment:
function duplicated. should this be extracted in a common file?

Reply:
Indeed, I've moved the function in the helpers/utils file